### PR TITLE
[Travis ] Add code coverage in Travis for Google Summer Code

### PR DESCRIPTION
**Pull request by kongtiaowang**
Add code coverage in Travis for google summer code

Code Coverage Report:        
  2019-05-28 18:39:01        
                             
 Summary:                    
  Classes:  4.08% (2/49)     
  Methods: 12.11% (81/669)   
  Lines:   20.96% (1452/6928)

### CODE REVIEW

**Comment by christinerogers** [comment]
Awesome, Thanks Shen @kongtiaowang !  
fyi: @Alexandralivadas @kchatpar

File: test/phpunit.xml [hunk 1]
```diff
@@ -62,11 +62,14 @@
             <!-- CodeCov crashes because installer/ has classes with the same
                  name as libraries/, so for now we only cover libraries and
                  exceptions under php/ -->
-            <directory suffix=".inc">../php/exceptions</directory>
+            <!-- In order to facilitate the calculation of the workload of the intern -->  
+            <!--directory suffix=".inc">../php/exceptions</directory-->
             <!--directory suffix=".php">../htdocs</directory-->
-            <directory suffix=".inc">../modules/*/php</directory>
+            <!--directory suffix=".inc">../modules/*/php</directory-->
```
Comment (driusan) [inline test/phpunit.xml]: Why are these directories being removed? It's also missing the src/ directory.

**Comment by christinerogers** [comment]
@kongtiaowang What should we do with this PR? 

If it's causing performance issues in Travis -- Can it be a script/tool that is optionally run? 

cc: @AlexandraLivadas @ridz1208 @johnsaigle

**Comment by kongtiaowang** [comment]
@driusan @christinerogers @ridz1208 @AlexandraLivadas 
This PR only to measure Alex's summer task (loris/php/libraries) . 
Since the code coverage is not important to Alex and our project, this PR should close.

**Comment by kongtiaowang** [comment]
No response, it means everyone agrees with me. I will close this PR.

**Pull request closed without merging**